src/Components/Table/DjTable.tsx

```diff
@@ -54,29 +55,35 @@ interface djRecords {
   totalCount: number
 }
 
+interface djFilter {
+  filteredValue: any
+  filtered: boolean
+  restriction: string
+}
+
 /**
  * DjTable component
  */
 export default class DjTable extends React.Component<
   DjTableProps,
   DjTableState
 > {
-  
   constructor(props: DjTableProps) {
     super(props)
     this.state = {
       data: { recordHeader: [], records: [[]], totalCount: 0 },
       dataAttributes: {
         attributeHeaders: [],
         attributes: { primary: [], secondary: [] },
-        unique_values: [[]]
+        unique_values: [[]],
```

Comment:
Whoops, looks like python's snake naming convention snuck in. :snake: 
```suggestion
        uniqueValues: [[]],
```